hs_core/views/__init__.py

```diff
@@ -96,11 +96,16 @@ def verify(request, *args, **kwargs):
 
 def change_quota_holder(request, shortkey):
     new_holder_uname = request.POST.get('new_holder_username', '')
```

Comment:
did you quickly search entire codebase for usages of change_quota_holder to ensure you know everything being impacted?

Reply:
Yes, the only other usages I saw were in some tests that I am now updating.